agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/MetricDataMapper.java

```diff
@@ -49,21 +52,27 @@
 
 public class MetricDataMapper {
 
+  private static final List<String> OTEL_PRE_AGGREGATED_METRIC_NAMES = new ArrayList<>();
```

Comment:
After being initialized, the only method called on `OTEL_PRE_AGGREGATED_METRIC_NAMES`  seems to be `contains`, so a `Set` may be more appropriate: 
`private static final Set<String> OTEL_PRE_AGGREGATED_METRIC_NAMES = new HashSet<>(4);`